articles/blablaconf-content-selection/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "BlablaConf: Why your talk might not be selected?"
+tags: ["conference", "blablaConf"]
+keywords: ["blablaConf", "Morocco", "Conferences", "Events"]
+pubDatetime: 2025-05-19
+authors: ["youssouf"] # the name of the author file
+slug: "blablaconf-content-selection-process"
+description: "Discover how the full BlablaConf process on selecting talks and speakers works"
+ogImage: "./images/blablaconf-cfp.png"
+---
+
+We all agree that content is one of the most important aspects of any conference, along with meeting new people and having a good time. A solid process for selecting content can significantly contribute to the success of any conference.
+
+Having organized the [BlablaConf](https://blablaconf.com) conference for five years now, we have developed a process that we believe works quite well.
+
+As transparency is one of our core values, this article serves to share our process with you. It will provide answers for those interested in submitting a talk in the future, as well as insights into why your talk may not have been selected in the past.
+
+This article only covers the content selection process for BlablaConf, if you are interested in other aspects of the conference feel free to mention it in the comments section and we will cover it in another article.
+
+## 🇲🇦 What is BlablaConf?
+
+For those who don't know, [BlablaConf](https://blablaconf.com) is our annual Geeksblabla community conference entirely in Moroccan Darija. It's a 5+1 days conference where speakers from various backgrounds and levels of expertise share their knowledge and experience with the community. Mostly online with few in-person days around Morocco.
```

Comment:
maybe elaborate more on the days. the 5+1 might be confusing. 5 (virtual) and (at least 1 in person) days